api/src/main/java/org/apache/cloudstack/query/QueryService.java

```diff
@@ -85,9 +85,12 @@
 public interface QueryService {
 
     // Config keys
-    static final ConfigKey<Boolean> AllowUserViewDestroyedVM = new ConfigKey<Boolean>("Advanced", Boolean.class, "allow.user.view.destroyed.vm", "false",
+    ConfigKey<Boolean> AllowUserViewDestroyedVM = new ConfigKey<>("Advanced", Boolean.class, "allow.user.view.destroyed.vm", "false",
             "Determines whether users can view their destroyed or expunging vm ", true, ConfigKey.Scope.Account);
 
+    ConfigKey<Boolean> SortKeyAscending = new ConfigKey<>("Advanced", Boolean.class, "sortkey.algorithm", "true",
```

Comment:
so true means that all lists for all users are sorted ascending? (if one of template, disk offering, service offering, network offering or zone)

Reply:
Yes @DaanHoogland . That's how global config seems to have been designed and used.

Reply:
Can you rectify the description to reflect the general use-case of the config key @anuragaw ?

Reply:
Fixing @rhtyd